modules/server/src/network/index.ts

```diff
@@ -6,6 +6,8 @@ import { fetchAllNodeAggregations } from './setup/query';
 import { createTypeDefs } from './typeDefs';
 import { NetworkConfig } from './types/setup';
 
+export let ALL_NETWORK_AGGREGATION_TYPES_MAP: Map<string, string> = new Map();
+
```

Comment:
runs on server startup. export so gql resolvers can quickly look up all [field:type]

Reply:
could also get it directly from resolver params but this is easier to parse

Reply:
I like the use of Map, useful definition lookup type.